tests/packagedcode/test_npm.py

```diff
@@ -367,6 +367,21 @@ def test_npm_yarn_with_package_json_resolve_dependencies(self):
             expected_file, result_file, remove_uuid=True, regen=REGEN_TEST_FIXTURES
         )
 
+    @pytest.fixture
+    def handler():
+        return NpmPackageLockJsonHandler()
+
+    def test_npm_package_lock_json_parse(handler):
+        test_file = 'npm/package-lock-v1/package-lock.json'
```

Comment:
This test already exists at https://github.com/aboutcode-org/scancode-toolkit/blob/0632016c3228481657acfc53cd1f318a1b102574/tests/packagedcode/test_npm.py#L254 :sweat_smile: